components/molecules/ContributorHoverCard/contributor-hover-card.tsx

```diff
@@ -1,63 +0,0 @@
-import { useRouter } from "next/router";
```

Comment:
I deleted this as it was a small component and it was only used in the hover-card-wrapper component. It also removed the need to prop drill another level.